README.md

```diff
@@ -1,28 +1,36 @@
 # Jurity: Fairness & Evaluation Library
 
-Jurity is a research library that provides fairness metrics, recommender system evaluations and bias mitigation techniques. The library adheres to PEP-8 standards and is tested heavily. 
+Jurity is a research library that provides classification metrics, fairness metrics, recommender system evaluations, and bias mitigation techniques. The library adheres to PEP-8 standards and is tested heavily.
```

Comment:
Not sure if classification needs to be added here. Fairness and reco should really be the focus. If we add it, it should be after reco system evals in the sentence.

Reply:
+1

Reply:
We discussed with @skadio that we will just mention we have the classification metrics as well